cpp/oneapi/dal/backend/primitives/reduction/test/reduction_dpc.cpp

```diff
@@ -34,6 +35,11 @@ namespace pr = oneapi::dal::backend::primitives;
 using reduction_types = std::tuple<std::tuple<float, sum<float>, square<float>>,
                                    std::tuple<double, sum<double>, square<double>>>;
 
+using finiteness_types = std::tuple<std::tuple<float, sum<float>, identity<float>>,
```

Comment:
is it necessary to have 
sum<float>, identity<float>>,
 sum<double>, identity<double> ?

Reply:
it'll be necessary for the two step process in checking finiteness, where the sum of the array is checked to be finite or not.  I can drop it if you want, but thought it would be good to have for the next step

Reply:
Okey, lets try it